pydarn/plotting/maps.py

```diff
@@ -71,12 +73,13 @@ def plot_mapdata(cls, dmap_data: List[dict], ax=None,
                      time_delta: float = 1,  alpha: float = 1.0,
                      len_factor: float = 150, color_vectors: bool = True,
                      cmap: str = None, colorbar: bool = True,
+                     contour_colorbar: bool = True,
                      colorbar_label: str = '', title: str = '',
                      zmin: float = None, zmax: float = None,
                      hmb: bool = True, boundary: bool = False,
                      radar_location: bool = False, map_info: bool = True,
                      imf_dial: bool = True, reference_vector: int = 500,
-                     projs: Projs = Projs.POLAR, **kwargs):
+                     projs: Projs = Projs.MAG, **kwargs):
```

Comment:
```suggestion
                     projs: Projs = Projs.POLAR, **kwargs):
```

Reply:
Looks like the code I was testing got accidentally pushed to the release branch. Oops.